I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 27, in script
    $ password == "Piss"
  File "game/script.rpy", line 27, in <module>
    $ password == "Piss"
NameError: name 'password' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 27, in script
    $ password == "Piss"
  File "renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 27, in <module>
    $ password == "Piss"
NameError: name 'password' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
TestGame 1.0
Thu May  5 21:01:12 2022
